frontend/components/bridge/BridgeTransferFlow.tsx

```diff
@@ -37,10 +37,10 @@ const TransferRow = ({ transfer }: { transfer: TokenTransfer }) => {
     <List.Item>
       <Flex justify="space-between" className="w-full">
         <Text>
-          {formatEther(fromAmount)} {fromSymbol}
+          {formatUnitsToNumber(fromAmount, 18, 8)} {fromSymbol}
```

Comment:
ideally we should pass decimals as they can vary (e.g. for USDC)
you can pass decimals in `updateCrossChainTransferDetails`, we have them in tokens
<img width="447" alt="image" src="https://github.com/user-attachments/assets/6543aeee-6634-45fd-8437-ec0a73a7a080" />